mail/tasks.py

```diff
@@ -0,0 +1,97 @@
+"""Tasks for mail app"""
+import logging
+from urllib.parse import urljoin
+
+from django.conf import settings
+from django.urls.base import reverse
+from celery import shared_task
+
+from mail import api
+from mail.models import NotificationEmail
+from ui.constants import VideoStatus
+
+
+log = logging.getLogger(__name__)
+
+
+def _get_recipients_for_video(video):
+    """
+    Returns a list of recipients that will receive notifications for a video.
+
+    NOTE: this needs to be updated to include all the admin moira lists associated to the collection
+    where the video is.
+
+    Args:
+        video (ui.models.Video): a video object
+
+    Returns:
+        list: a list of strings representing emails
+    """
+    recipients_list = []
+    owner_email = video.collection.owner.email
+    if owner_email:
+        recipients_list.append(owner_email)
+    return recipients_list
+
+
+STATUS_TO_NOTIFICATION = {
+    VideoStatus.COMPLETE: NotificationEmail.SUCCESS,
+    VideoStatus.TRANSCODE_FAILED_INTERNAL: NotificationEmail.OTHER_ERROR,
+    VideoStatus.TRANSCODE_FAILED_VIDEO: NotificationEmail.INVALID_INPUT_ERROR,
+    VideoStatus.UPLOAD_FAILED: NotificationEmail.OTHER_ERROR,
+}
+
+
+def send_notification_email(video):
```

Comment:
I decided to put them here because `mail.api` contains just the Mailgun client.
We might move them to an `task_api` module if you feel strong about that